src/ImageSharp/Formats/Jpeg/JpegMetadata.cs

```diff
@@ -23,12 +26,46 @@ private JpegMetadata(JpegMetadata other)
         {
             this.Quality = other.Quality;
             this.ColorType = other.ColorType;
+
+            this.LumaQuantizationTable = other.LumaQuantizationTable;
+            this.ChromaQuantizationTable = other.ChromaQuantizationTable;
+            this.LumaQuality = other.LumaQuality;
+            this.ChromaQuality = other.ChromaQuality;
         }
 
+        /// <summary>
+        /// Gets or sets luminance qunatization table derived from jpeg image.
+        /// </summary>
+        /// <remarks>
+        /// Would be null if jpeg was encoded using table from ITU spec
+        /// </remarks>
+        internal Block8x8? LumaQuantizationTable { get; set; }
+
+        /// <summary>
+        /// Gets or sets chrominance qunatization table derived from jpeg image.
+        /// </summary>
+        /// <remarks>
+        /// Would be null if jpeg was encoded using table from ITU spec
+        /// </remarks>
+        internal Block8x8? ChromaQuantizationTable { get; set; }
+
+        internal double LumaQuality { get; set; }
+
+        internal double ChromaQuality { get; set; }
+
         /// <summary>
         /// Gets or sets the encoded quality.
         /// </summary>
-        public int Quality { get; set; } = 75;
+        public int Quality
+        {
+            get => (int)Math.Round(this.LumaQuality);
+            set => this.LumaQuality = value;
+        }
+
+        /// <summary>
+        /// Gets a value indicating whether jpeg was encoded using ITU section spec K.1 quantization tables
+        /// </summary>
+        public bool ItuSpecQuantization => !this.LumaQuantizationTable.HasValue && !this.ChromaQuantizationTable.HasValue;
```

Comment:
Naming suggestion:

```suggestion
        public bool UsesItuSpecQuantization => !this.LumaQuantizationTable.HasValue && !this.ChromaQuantizationTable.HasValue;
```

Reply:
I decided to drop it in favour of internal check if tables are carried from decoding routine, this is a very specific check which I believe 99.99% users won't bother with.